buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -179,6 +179,12 @@ public void run() {
     public static final PooledByteBufAllocator DEFAULT =
             new PooledByteBufAllocator(PlatformDependent.directBufferPreferred());
 
+    /**
+     * Default instance which uses fallback policy.
+     */
+    public static final FallbackByteBufAllocator FALL_BACK =
+            new FallbackByteBufAllocator(new PooledByteBufAllocator(PlatformDependent.directBufferPreferred()));
```

Comment:
I think this should wrap the `DEFAULT` instance, as otherwise there would be two independent pools.